src/App.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
 import {createBrowserRouter, RouterProvider} from 'react-router-dom';
-import TeamOverview from './pages/TeamOverview';
-import Teams from './pages/Teams';
-import UserOverview from './pages/UserOverview';
+import styled from 'styled-components';
+
+import {TeamOverview, Teams, UserOverview} from './pages';
+
+const Container = styled.div`
```

Comment:
With this change I remove the need to add the Container component on the 3 pages (Teams, TeamOverview and UserOverview)